src/Sylius/Bundle/AdminBundle/Resources/config/grids/order.yml

```
@@ -77,6 +77,15 @@ sylius_grid:
                     label: sylius.ui.currency
                     sortable: ~
             filters:
+                state:
+                    type: select
+                    label: sylius.ui.state
+                    form_options:
+                        choices:
+                            sylius.ui.cart: cart
+                            sylius.ui.cancelled: cancelled
+                            sylius.ui.new: new
+                            sylius.ui.fulfilled: fulfilled
```

```suggestion
                            sylius.ui.new: new
                            sylius.ui.cancelled: cancelled
                            sylius.ui.fulfilled: fulfilled
```
Can you even see carts?